build_web_compilers/lib/builders.dart

```diff
@@ -51,11 +52,16 @@ Builder ddcKernelBuilder(BuilderOptions options) {
       experiments: _readExperimentOption(options));
 }
 
-Builder sdkJsCopyBuilder(_) => SdkJsCopyBuilder();
-PostProcessBuilder sdkJsCleanupBuilder(BuilderOptions options) =>
-    FileDeletingBuilder(
-        ['lib/src/dev_compiler/dart_sdk.js', 'lib/src/dev_compiler/require.js'],
-        isEnabled: options.config['enabled'] as bool ?? false);
+// Copies the `require.js` file from the sdk into this package.
```

Comment:
[nit] Is this comment necessary? Should it be a doc comment? Should it mention the other JS files?

Reply:
I just removed it